db/migrate/20150910170527_add_data_to_credit_card.rb

```diff
@@ -1,5 +1,9 @@
 class AddDataToCreditCard < ActiveRecord::Migration
   def change
-    add_column :spree_credit_cards, :data, :json
+    if ActiveRecord::Base.connection.adapter_name != 'SQLite' && ActiveRecord::Base.connection.supports_json?
```

Comment:
@dfranciosi We should make sure to update the rails dependency then, and to cut a new version of the gem.